### PR TITLE
docs: migrated content for NMNet and user-2 from Network to subfolders

### PR DESCRIPTION
**Pull request by ayushgml**
This issue fixes:
- #1815 

The folder structure now look like this:

Screenshot 2024-05-29 at 11 40 15 AM

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
Needs rebase

File: website/content/en/docs/Config/Network/_index.md [hunk 1]
```diff
@@ -4,17 +4,20 @@ weight: 30
...
 See the following flowchart to choose the best network for you:
+
 ```mermaid
 flowchart
   connect_to_vm_via{"Connect to the VM via"} -- "localhost" --> default["Default"]
-  connect_to_vm_via -- "IP" --> connect_from{"Connect to the VM IP from"}
-  connect_from -- "Host" --> vm{"VM type"}
+  connect_to_vm_via -- "IP" --> connect_from{"Connect to the VM IP for"}
+  connect_from -- "host-to-guest \n communication" --> vm{"VM type"}
   vm -- "vz" --> vzNAT["vzNAT"]
```
Comment (AkihiroSuda) [inline website/content/en/docs/Config/Network/_index.md:13]: ?

**Comment by ayushgml** [comment]
@AkihiroSuda I have made the changes and rebase. Can you please update me if any more changes/improvements are required?

File: website/content/en/docs/Config/Network/VMNet/_index.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+#
+# None of the paths segments may be symlinks, why it has to be /private/var
+# instead of /var etc.
+paths:
+# socketVMNet requires Lima >= 0.12 .
+# socketVMNet has precedence over vdeVMNet.
+  socketVMNet: /opt/socket_vmnet/bin/socket_vmnet
+# vdeSwitch and vdeVMNet are DEPRECATED.
+  vdeSwitch: /opt/vde/bin/vde_switch
+  vdeVMNet: /opt/vde/bin/vde_vmnet
```
Comment (AkihiroSuda) [inline website/content/en/docs/Config/Network/VMNet/_index.md:59]: Seems copied from an old revision

**Review by AkihiroSuda — CHANGES_REQUESTED** [review]
https://github.com/lima-vm/lima/pull/2375/files#r1623234867

**Comment by alexandear** [comment]
@ayushgml, could you please update your PR in line with the comments provided above? Thank you!

**Comment by AkihiroSuda** [comment]
Closing, but feel free to refresh and reopen

**Pull request closed without merging**